generate-commit.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -ex
+echo "Pulling meta-balena"
+
+if [ -z $TAG ]; then
+	echo "A \$TAG variable is required with the new supervisor version"
+	exit 1
+fi
+
+if [ ! -d /tmp/meta-balena ]; then
+	git clone git@github.com:balena-os/meta-balena /tmp/meta-balena
+else
+	cd /tmp/meta-balena
+	git reset --hard
```

Comment:
also `git checkout master` before and `git pull` after, to work on the latest version?

Reply:
We do that after `development`, as that's what we merge the changes onto.